App.vue

```diff
@@ -1,38 +1,17 @@
 <template>
   <div id="app">
-    <component :is="layout">
+    <DefaultLayout>
       <router-view />
-    </component>
+    </DefaultLayout>
   </div>
 </template>
 
 <script>
-import { mapState } from 'vuex';
-const DefaultLayout = () =>
-  import(/* webpackChunkName: "vsf-layout-default" */ './layouts/Default');
-const EmptyLayout = () =>
-  import(/* webpackChunkName: "vsf-layout-empty" */ './layouts/Empty');
-const MinimalLayout = () =>
-  import(/* webpackChunkName: "vsf-layout-minimal" */ './layouts/Minimal');
+import DefaultLayout from './layouts/Default'
 
 export default {
   components: {
-    DefaultLayout,
-    EmptyLayout,
-    MinimalLayout
-  },
-  data () {
-    return {
-      ordersData: []
-    };
-  },
-  computed: {
-    ...mapState({
-      overlayActive: state => state.ui.overlay
-    }),
-    layout () {
-      return `${this.$route.meta.layout || 'default'}-layout`;
-    }
+    DefaultLayout
   }
 };
 </script>
```

Comment:
@gibkigonzo why components like `ProductListing.vue` or `Tooltip.vue` was deleted in this PR?

Reply:
we don't import those components anymore in any other component